bootstrap/src/lib/metrics/index.ts

```diff
@@ -0,0 +1,57 @@
+import * as client from 'prom-client'
+
+client.collectDefaultMetrics()
+client.register.setDefaultLabels(
+  // we'll inject both name and versions in
+  // when EAEE gets merged, because it'll be a lot easier
+  // to refactor with full type coverage support
+  { app_name: process.env.METRICS_NAME || 'N/A', app_version: 'N/A' },
+)
+export const METRICS_ENABLED = !!process.env.EXPERIMENTAL_METRICS_ENABLED
```

Comment:
There is a `utils/parseBool` that you can consider using here. 
Just so someone would not set an explicit `EXPERIMENTAL_METRICS_ENABLED=false`.

Reply:
Sounds good!